scripts/npcs/casern_cellar/guarddain.lua

```diff
@@ -0,0 +1,73 @@
+--[[
+
+  Guard Dain
+
+  Copyright (C) 2014 Jessica Beller
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--]]
+
+local function guard_talk(npc, ch)
+    local function deliver_food()
+        say("Or do you have the food? Excellent! I was getting hun... "
+            .."I mean, you can leave the rations here with me, I will take care of bringing it to the prisoners.")
+        local choices = {
+            "I will give it to them myself.",
+            "Sure, here it is."
+        }
+        local res = ask(choices)
+        if (res == 1) then
+            say("Fine. Go ahead, just follow the stink, you can't miss it.")
+        else
+            local bread = ch:inv_count("Bread")
+            if (bread >= GUARDDUTY_BREADAMOUNT) then
+                ch:inv_change("Bread", -GUARDDUTY_BREADAMOUNT)
+                chr_set_quest(ch, "soldier_goldenfields_guardduty", "deliveredfood")
+                ch:set_questlog_description(QUESTID_GODWIN_GUARDDUTY,
+                    "Talk to Prisoner Asher in the casern prison cell.", true)
+                say("Great! Now you can go upst-")
```

Comment:
I would exchange the two lines above. Once we add notifications for the questlog it would spoiler the next action I guess :)

If you swap it it would show the notification together with the line below.